src/NetscriptFunctions.js

```diff
@@ -1619,7 +1619,7 @@ function NetscriptFunctions(workerScript) {
         },
         getBitNodeMultipliers: function() {
             updateDynamicRam("getBitNodeMultipliers", getRamCost("getBitNodeMultipliers"));
-            if (SourceFileFlags[5] <= 0) {
+            if (Player.bitNodeN !== 5 && SourceFileFlags[5] <= 0) {
```

Comment:
Seems weird that the check is flipped compared to the one in getServer (line 1630) but okay